### DISCUSSION TITLE
[CICD] 젠킨스로 간단한 CICD 돌려보기

### DISCUSSION THREAD

**Discussion by 02ggang9** [BDD-CLUB/challenge-every-week-posting#414] [1. 개발 관련 글 모음]
졸과에서는 github action 말고 젠킨스를 써 보고 싶어서 한번 공부하고 있습니다..

https://02ggang9.github.io/infra/Jenkins1/

**Comment by jojaeng2** [comment]
재밌게 잘 읽었습니다. 
CI의 과정만 존재하는데 CD는 어떻게 구성하실지 궁금하네요
